src/utils.ts

```diff
@@ -80,7 +80,7 @@ export const problemDescription = (problem: Problem): string => {
 // ══════════════════════════════════════════════════════════════════════════════════════════════════════════════════
 
 export const publicPath = (...paths: string[]): string => {
-  return path.join(__dirname, '..', 'public', ...paths)
+  return path.join(__dirname, '../..', 'public', ...paths)
```

Comment:
Ojo con las barras acá (que cambian entre Windows y el resto), `path.join` está para no tener que lidiar con ellas.

Creo que esto debería ser:
```suggestion
  return path.join(__dirname, '..', '..', 'public', ...paths)
```

Reply:
aaaah, piola!!! no lo sabía. Justo ayer veía que estábamos haciendo `\n` en la consola y pensaba: en Windows seguro que quiere además un `\r`.